<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ttv</class>
 <widget class="QMainWindow" name="ttv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ttviewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="preference_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Frame Preferences</string>
    </property>
    <widget class="QSpinBox" name="width_spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>60</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="height_spinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>60</number>
     </property>
     <property name="maximum">
      <number>720</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="num_serv_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Streams</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="width_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Display Width</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="height_lineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Height</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rows_wanted_spinBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>48</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rows Wanted</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="titleSpace_spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Title Space</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Streams Available</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ffplay_pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FFplay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setMPVs_pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>MPV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>LUF_mon</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
